proto/plan.proto

```diff
@@ -36,20 +36,21 @@ message Type {
 // double, use that, otherwise store a string representation.
 message Const {
 	bool isnull			= 1;
+	bool isMaxValue = 2;
```

Comment:
I don't think isMaxValue be a field of Const is a good choice.
That will make constant fold complicated。

Reply:
OK, I will add a new expression to handle ‘MAXVALUE’